test/tools/ossfuzz/abiV2FuzzerCommon.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <libsolidity/interface/CompilerStack.h>
+#include <libyul/AssemblyStack.h>
+#include <liblangutil/Exceptions.h>
+#include <liblangutil/SourceReferenceFormatter.h>
+#include <libdevcore/Keccak256.h>
+
+namespace dev
+{
+namespace test
+{
+namespace abiv2fuzzer
+{
+class SolidityExecutionFramework
+{
+public:
+	SolidityExecutionFramework();
+	SolidityExecutionFramework(langutil::EVMVersion _evmVersion);
+
+	Json::Value getMethodIdentifiers()
+	{
+		return m_compiler.methodIdentifiers(m_compiler.lastContractName());
+	}
+
+	dev::bytes compileContract(std::string const& _sourceCode)
+	{
+		// Silence compiler version warning
+		std::string sourceCode = "pragma solidity >=0.0;\n";
+		sourceCode += "pragma experimental ABIEncoderV2;\n";
+		sourceCode += _sourceCode;
+		m_compiler.setSources({{"", sourceCode}});
+		m_compiler.setEVMVersion(m_evmVersion);
+		m_compiler.setOptimiserSettings(m_optimiserSettings);
+		if (!m_compiler.compile()) {
+			langutil::SourceReferenceFormatter formatter(std::cerr);
+
+			for (auto const& error: m_compiler.errors())
+				formatter.printExceptionInformation(
+						*error,
+						(error->type() == langutil::Error::Type::Warning) ? "Warning" : "Error"
```

Comment:
I think there is a wrapper now that does this for you.

Reply:
done